### PR TITLE
feat(examples): Update emotion

### PR DESCRIPTION
**Pull request by LekoArts**
- Update the dependencies
- Change all imports to `react-emotion`
- Fix eslint errors (e.g. PropTypes)
- Remove stray backticks
- Only use injectGlobal once (not like in the PrismJS styles which is very chaotic)

I couldn't find something online if [this](https://www.styled-components.com/docs/api#deprecated-injectglobal) also applies to emotion, but I think it's a good change nevertheless:

> We do not encourage the use of this. Try to use it once per app at most, if you must, contained in a single file.

### CODE REVIEW

**Comment by ThomOrlo** [comment]
> I couldn't find something online if [this](https://www.styled-components.com/docs/api#deprecated-injectglobal) also applies to emotion, but I think it's a good change nevertheless:

@DSchau same as styled-components 4, with Emotion 10 :
`import { Global, css } from '@emotion/core'`

https://github.com/emotion-js/emotion/issues/637
https://next.emotion.sh/docs/globals

**Comment by LekoArts** [comment]
I was referring to the quote of injectGlobal 😅
Not emotion 10

File: examples/using-emotion-prismjs/src/components/layout.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ const link = css`
   color: inherit;
 `
 
-class Layout extends React.Component {
+class PureLayout extends React.Component {
```
Comment (DSchau) [inline examples/using-emotion-prismjs/src/components/layout.js]: Why the change?

Comment (LekoArts) [reply examples/using-emotion-prismjs/src/components/layout.js]: eslint wanted an explicite name (not `export default props`) and since I can't name them both `Layout` I gave this one another name.

Comment (DSchau) [reply examples/using-emotion-prismjs/src/components/layout.js]: I guess I would've just called it BaseLayout or something! Pure implies (to me!) something like a PureComponent i.e. a component that doesn't re-render when its props are shallowly equal.

Comment (LekoArts) [reply examples/using-emotion-prismjs/src/components/layout.js]: And ai chose Pure because it’s without GraphQL 😀 But I don’t care really

Comment (DSchau) [reply examples/using-emotion-prismjs/src/components/layout.js]: Yeah, not a big deal either way, _especially_ because this is an example. Let's just merge :) Thank you!
